apps/wallet/src/background/keyring/Vault.ts

```diff
@@ -10,32 +12,42 @@ import {
     validateEntropy,
 } from '_shared/utils/bip39';
 
-export const LATEST_VAULT_VERSION = 1;
+import type { ExportedKeypair, Keypair } from '@mysten/sui.js';
+
+export const LATEST_VAULT_VERSION = 2;
```

Comment:
We might want to start investing a little more in the upgrade path for stored data, I think there's probably some opportunity here to make this better / more generic.

Reply:
Yeah I agree we need to add some kind of upgrade/migrate stored data, this is probably a bit more tricky I think, since the data it encrypted and we can run migrations only when we have the password but we can do it. Although I would prefer to do it separately. Since it will be useful in other places as well but will need some refactoring to stop using storage randomly from different places and have some kind of util/lock that will handle storage access